doc/source/whatsnew/v1.2.0.rst

```diff
@@ -195,7 +195,7 @@ Deprecations
 ~~~~~~~~~~~~
 - Deprecated parameter ``inplace`` in :meth:`MultiIndex.set_codes` and :meth:`MultiIndex.set_levels` (:issue:`35626`)
 - Deprecated parameter ``dtype`` in :~meth:`Index.copy` on method all index classes. Use the :meth:`Index.astype` method instead for changing dtype(:issue:`35853`)
--
+- Date parser functions :func:`parse_date_time`, :func:`parse_date_fields`, :func:`parse_all_fields` and :func:`generic_parser` from ``pandas.io.date_converters`` are deprecated and will be removed in a future version; use pandas.to_datetime() instead (GH35741)
```

Comment:
can you put in ```:func:`to_datetime` ``` so it renders all of the others won't render at all i think (e.g. ```:func:`parse_date_time` ```. so those should be updated to a fully qualified path (e.g. pandas.io.date_converters)